broker-daemon/index.js

```diff
@@ -253,17 +253,8 @@ class BrokerDaemon {
    * for a period of time, until it is either successful or there is actually something wrong.
    * @returns {void}
    */
-  async validateEngines () {
-    this.engines.forEach(async (engine, symbol) => {
-      try {
-        await exponentialBackoff(() => { return engine.validateNodeConfig() }, EXPONENTIAL_BACKOFF_ATTEMPTS, EXPONENTIAL_BACKOFF_DELAY, {symbol})
-      } catch (e) {
-        this.logger.error(`Failed to validate engine for ${symbol}, error: ${e}`)
-        return
-      }
-
-      this.logger.info(`Validated engine configuration for ${symbol}`)
-    })
+  validateEngines () {
```

Comment:
do we need to validate engines separately, or can we do it while we are creating the engines?

Reply:
@treygriffith having it separated is easier to read. When we assign `engines` we create them and map them to `this.engines` and it felt weird to validate inside of the map.